velox/row/tests/CMakeLists.txt

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 
 add_executable(
-  velox_row_test UnsafeRowSerializerTest.cpp UnsafeRowDeserializerTest.cpp
```

Comment:
Did we not have UnsafeRowBatchDeserializerTest in the cmake build file before? Perhaps, mention this fix in the PR description as well.

Reply:
Not before, right